gitlab-pipeline/stage/release.yml

```diff
@@ -53,6 +53,23 @@ release_docker_images:manual:
   extends: .template_release_docker_images
 
 
+release_docker_multiplatform_images:manual:
+  when: manual
+  extends: .template_release_docker_images
+  image: "registry.gitlab.com/northern.tech/mender/mender-test-containers:docker-multiplatform-buildx-v1-master"
+  script:
+    # for pre 2.4.x releases, omit --version-type
+    - if $WORKSPACE/integration/extra/release_tool.py --help | grep -e --version-type; then
+    -   VERSION_TYPE_PARAMS="--version-type docker"
+    - fi
+    # Load, tag and push Docker images
+    - for image in $($WORKSPACE/integration/extra/release_tool.py --list docker); do
+        version=$($WORKSPACE/integration/extra/release_tool.py --version-of $image $VERSION_TYPE_PARAMS --in-integration-version $INTEGRATION_REV);
+        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $image docker_url);
+        cicd_tag="${image^^}_REV"
+        regctl image copy registry.gitlab.com/northern.tech/mender/${!cicd_tag}:${version} $docker_url:${version};
```

Comment:
As per today CI pipelines design, this won't work for Git tags because the tags are no published multi-platfrom from individual pipelines. We've been tagging from the `.x` instead.

Instead of modifying now the template to publish on tags (in light of the rework we are doing for the monorepo, and to avoid unforeseen consequence for the current release process) we can use here the `${image}_REV_GIT_SHA` variables to get the source to tag from. That will be much safer than using `${image}_REV` which might be a branch.

To use this new set of variables we need the init_workspace artifacts and environment.